setup.cfg

```diff
@@ -0,0 +1,78 @@
+[metadata]
+name = nibabel
+url = https://nipy.org/nibabel
+download_url = https://github.com/nipy/nibabel
+author = nibabel developers
+author_email = neuroimaging@python.org
+maintainer = Chris Markiewicz
+maintainer_email = neuroimaging@python.org
+classifiers =
+    Development Status :: 4 - Beta
+    Environment :: Console
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3.4
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Topic :: Scientific/Engineering
+license = MIT License
+description = Access a multitude of neuroimaging data formats
+long_description = file:README.rst
+long_description_content_type = text/x-rst; charset=UTF-8
+platforms = OS Independent
+provides =
+    nibabel
+    nisext
+
+[options]
+python_requires = >=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*
+install_requires =
+    numpy >=1.8
+    six >=1.3
+    bz2file ; python_version < "3.0"
```

Comment:
Do you by any chance remember from what setuptools version this extended setup.cfg is supported? I just fear possible troubles on older systems

Reply:
30.3.0

Reply:
In general, will it be a problem for older systems to upgrade setuptools? That version had a `python_requires='>=2.6,!=3.0.*,!=3.1.*,!=3.2.*'` constraint, which is a superset of our own support level.

Reply:
@yarikoptic Sorry to bug you, but I just want to check whether that's sufficiently old, or you still think this could produce problems... I can revert to a pre-30.3-style setuptools config, if this is a serious concern.